src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.helper;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.CcdCollectionElement;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.CcdDocument;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.ScannedDocument;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Document;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toList;
+
+public class ScannedDocumentsHelper {
+
+    private static final ObjectMapper objectMapper;
+
+    static {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
+    }
+
+    private ScannedDocumentsHelper() {
+        // utility class
+    }
+
+    @SuppressWarnings("unchecked")
+    public static List<ScannedDocument> getScannedDocumentsForExceptionRecord(CaseDetails caseDetails) {
+        List<Map<String, Object>> data = (List<Map<String, Object>>) caseDetails.getData().get("scanRecords");
```

Comment:
@nitinprabhu do we need different property names in exception records and supplementary evidence (`scanRecords` and `scannedDocuments`) or could we pick one name and go with that.

@aliveni don't do that in this PR though, unrelated change, just would be good to fix it

Reply:
@timja I don't think so we need different names. Yes can be clubbed into one 👍